CMakeLists.txt

```diff
@@ -37,3 +43,9 @@ target_include_directories(AsyncRgbLedAnalyzer PRIVATE ${ANALYZER_SDK_ROOT}/incl
 
 # TODO - define installation dir
 #install(TARGETS AsyncRgbLedAnalyzer RUNTIME DESTINATION ${ANALYZER_PLUGIN_DIR})
+
+# for AStyle
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+# FlightGear packaging (to build a source tarball)
```

Comment:
This is an exciting new direction for the project! :)

Reply:
Whoops, ahem, I'll remove that :)